pkg/kv/kvserver/replica_raft.go

```diff
@@ -1306,11 +1313,16 @@ func (r *Replica) refreshProposalsLocked(
 		// so delays shouldn't dramatically change the detection latency.
 		inflightDuration := r.store.cfg.RaftTickInterval * time.Duration(r.mu.ticks-p.createdAtTicks)
 		if ok && inflightDuration > slowReplicationThreshold {
+			slowProposalCount++
 			if maxSlowProposalDuration < inflightDuration {
 				maxSlowProposalDuration = inflightDuration
 				maxSlowProposalDurationRequest = p.Request
-				slowProposalCount++
 			}
+		} else if inflightDuration > time.Minute {
```

Comment:
Should we have a constant for this? No strong opinion either way.

Reply:
Is there already some default for `slowReplicationThreshold` above? Could reuse it here.